stdlib/2and3/builtins.pyi

```diff
@@ -250,10 +250,10 @@ class float:
 
     def __eq__(self, x: object) -> bool: ...
     def __ne__(self, x: object) -> bool: ...
-    def __lt__(self, x: float) -> bool: ...
-    def __le__(self, x: float) -> bool: ...
-    def __gt__(self, x: float) -> bool: ...
-    def __ge__(self, x: float) -> bool: ...
+    def __lt__(self, x: Union[int, float]) -> bool: ...
+    def __le__(self, x: Union[int, float]) -> bool: ...
+    def __gt__(self, x: Union[int, float]) -> bool: ...
+    def __ge__(self, x: Union[int, float]) -> bool: ...
```

Comment:
This change is unnecessary, since `float` in argument types also accepts `int`s per PEP 484.

Reply:
Thanks, I missed that comment in PEP 484.